test/index.js

```diff
@@ -400,6 +400,15 @@ describe('Test stream events', () => {
     const data = removeEventData(converter(insertEvent.Records[0].dynamodb.NewImage))
     assert.deepEqual(data, body._source)
   })
+
+  it('MODIFY: should remove deleted fields', async () => {
+    await pushStream({ event: modifyEventWithDeletedField, index: INDEX, endpoint: ES_ENDPOINT })
```

Comment:
Hi,

Thanky you for fast PR.
I guess `useBulk` should be set here to `true`?

Reply:
You're right, my mistake. Just added proper test.